docs/source/getting-started.md

```diff
@@ -0,0 +1,110 @@
+| [boa3.builtin.math](https://dojo.coz.io/neo3/boa/boa3/builtin/boa3-builtin-math.html)                                                | a small sample of functions similar to Python's math.                                                                                   | [sqrt](https://dojo.coz.io/neo3/boa/boa3/builtin/boa3-builtin-math.html#boa3.builtin.math.sqrt), [floor](https://dojo.coz.io/neo3/boa/boa3/builtin/boa3-builtin-math.html#boa3.builtin.math.floor), [ceil](https://dojo.coz.io/neo3/boa/boa3/builtin/boa3-builtin-math.html#boa3.builtin.math.ceil)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+
+
+### Hello World
+Let's write a quick Hello World script that has a method that will save `"Hello World"` on the blockchain 
+and another method that will return the string.
+
+Those 2 functions will need to be callable and will also need to change the values inside the storage, 
+so let's import both the `public` decorator and the `storage` package.
+
+```python
+# hello_world.py
+from boa3.builtin.compile_time import public
+from boa3.builtin.interop import storage
+
+
+@public     # the public decorator will make this method callable
+def save_hello_world():             # an empty return type indicates that the return is None
+    storage.put("first script", "Hello World")      # the put method will store the "Hello World" value with the "first script" key
+
+
+@public     # the public decorator will make this method callable too
+def get_hello_world() -> str:       # this method will return a string, so it needs to specify it
+    return storage.get("first script")              # the get method will return the value associated with "first script" key
+```
+
+Alternatively, instead of creating a `save_hello_world` function and having to call it yourself, you could instead create
```

Comment:
I don't think we should say it is "Alternatively", because you can do both on the same contract. Let's use this as a seconds step of the tutorial with a new title